README.md

```diff
@@ -16,16 +16,13 @@ Requirements
 Installation
 ------------
 
-This package is available on the Python Package Index (PyPI), and can be
-installed with pip:
+This application can be installed by following the steps below:
```

Comment:
I think we should go back to referring to this as a "package", not an "application".